packages/synapse-react-client/src/components/GoalsV2/GoalsV2.tsx

```diff
@@ -101,10 +101,12 @@ export const GoalsV2: React.FC<GoalsV2Props> = (props: GoalsV2Props) => {
       }
     }) ?? []
 
+  const ContentComponent = showDesktop ? GoalsV2Desktop : GoalsV2Mobile
+
   return (
     <Box
       sx={{
-        height: '560px',
+        minHeight: '560px',
```

Comment:
In mobile, the content could expand beyond 560px and would create a scrollbar with a constrained height--this should be `min-height` to prevent that.